osier/equations.py

```diff
@@ -0,0 +1,160 @@
+from abc import ABC, abstractmethod
+import numpy as np
+
+class OsierEquation(ABC):
+
+    def __init__(self) -> None:
+        pass        
+
+    @abstractmethod
+    def _do(self, technology_list, X):
+        pass
```

Comment:
Consider adding a docstring to this function so its purpose is more clear.

Reply:
This whole class is unused for the moment. I'm debating whether or not I want the objectives and constraints to be classes or functions. I went with classes for now since the implementation is much more straight forward. I could easily delete this.